cfg/config.go

```diff
@@ -57,7 +63,11 @@ func GetConfigAndValidate() (*Config, error) {
 	if config.KinesisEndpoint == "" && config.PusherMode == KINESIS_PUSHER {
 		return nil, errors.New("KINESIS_ENDPOINT must be set as environment variable when PUSHER_MODE is set to kinesis")
 	}
-
+	region := awsRegion
+	if region == "" {
+		region = awsDefaultRegion
+	}
```

Comment:
I am not sure why this is added, do you think of case like lambda deployed to some region and it needs to connect to another region for using another services?

Reply:
here's the AWS documentation:  https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html

AWS_DEFAULT_REGION – The default AWS Region where the Lambda function is executed.

AWS_REGION – The AWS Region where the Lambda function is executed. If defined, this value overrides the AWS_DEFAULT_REGION.

Reply:
In which case our extension needs this override?

Reply:
removed AWS_DEFAULT_REGION